### PR TITLE
Updated Bengali Linux Courses

### PR DESCRIPTION
**Pull request by apu52**
## What does this PR do?
Add resource(s) 

### Description
I have added 2 Linux courses in Bengali which are very useful 
### Why is this valuable (or not)?
It is valuable
### How do we know it's really free?
As the whole playlist is available in YouTube . It's totally free.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course
## Checklist:
- [ x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by apu52** [comment]
@Ajay-singh1 @eshellman Please take a review of my PR & please merge it.

**Review by Ajay-singh1 — APPROVED** [review]
LGTM!

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -193,6 +193,8 @@
 
 ### Linux
 
+* [Kali Linux Basics Full Course In Bangla By Mehedi Shakeel](https://www.youtube.com/playlist?list=PL-8coCSOYV9F_tVxJSX1tzNram6PH0K4O) - Mehedi Shakeel
+*  [Linux Essentials Full course](https://www.youtube.com/playlist?list=PLzOdtYcAxAiOzVwsu7diaDMJiWlbDPiTb) - CSLCBT - Bangla
```
Comment (eshellman) [inline courses/free-courses-bn.md:197]:
```suggestion
* [Linux Essentials Full course](https://www.youtube.com/playlist?list=PLzOdtYcAxAiOzVwsu7diaDMJiWlbDPiTb) - CSLCBT - Bangla
```

**Comment by apu52** [comment]
@eshellman I have updated the changes in the PR. Please take a review of my PR & please merge it.

**Comment by apu52** [comment]
@Ajay-singh1 @sohansai @eshellman I have updated the changes in the PR. Please take a review of my PR & please merge it.

**Comment by Ajay-singh1** [comment]
@apu52 sync your branch with the main branch.

**Comment by apu52** [comment]
@Ajay-singh1 I have synced/updated my branch with the main branch

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -194,6 +194,8 @@
 
 ### Linux
 
+* [Kali Linux Basics Full Course In Bangla By Mehedi Shakeel](https://www.youtube.com/playlist?list=PL-8coCSOYV9F_tVxJSX1tzNram6PH0K4O) - Mehedi Shakeel
+* [Linux Essentials Full course](https://www.youtube.com/playlist?list=PLzOdtYcAxAiOzVwsu7diaDMJiWlbDPiTb) - CSLCBT - Bangla
```
Comment (eshellman) [inline courses/free-courses-bn.md:198]:
```suggestion
* [Linux Essentials Full course](https://www.youtube.com/playlist?list=PLzOdtYcAxAiOzVwsu7diaDMJiWlbDPiTb) - CSLCBT Bangla
```
make sure search parser doesn't get confused

**Comment by apu52** [comment]
@eshellman I have resolved the changes. Can you please take a review & merge it.

**Pull request merged**